frontend/i18n/en/rules.js

```diff
@@ -0,0 +1,38 @@
+export default {
+  colorRules: {
+    colorRequired: 'Color is required'
+  },
+  labelNameRules: {
+    labelRequied: 'Label name is required',
```

Comment:
Typo: `Requied` -> `Required`

Reply:
Addressed in https://github.com/doccano/doccano/pull/976/commits/484ce03323a7ded5190dc6c8c425735c0a765b19